pkg/api/team.go

```diff
@@ -4,19 +4,38 @@ import (
 	"github.com/grafana/grafana/pkg/api/dtos"
 	"github.com/grafana/grafana/pkg/bus"
 	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/teamguardian"
 	"github.com/grafana/grafana/pkg/util"
 )
 
 // POST /api/teams
-func CreateTeam(c *m.ReqContext, cmd m.CreateTeamCommand) Response {
+func (hs *HTTPServer) CreateTeam(c *m.ReqContext, cmd m.CreateTeamCommand) Response {
 	cmd.OrgId = c.OrgId
+
+	if c.OrgRole == m.ROLE_VIEWER {
```

Comment:
Maybe add test or comment that says something like `// viewers have access to the teams group in the api but should not be able to create teams`

Reply:
Might be better to solve it in api.go instead, not sure.